test_BMS_datastream.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import bms_input_handler
+
+
+class test_streaming_bms_data(unittest.TestCase):
+
+    def test_bms_param_generator(self):
+        bms_param = bms_input_handler.generate_bms_parameters()
+        self.assertTrue(bms_param is not None)
+
+    def test_bms_param_within_limits(self):
+        bms_param = bms_input_handler.generate_bms_parameters()
+        self.assertTrue('temperature', bms_param['temperature'])

```

Comment:
assertTrue takes a boolean expression and a message. In this assert, `'temperature'` always evaluates to `True` and `bms_param['temperature']` is the message which will never be displayed.
This test never fails. It's a good practice to make the test fail once to ensure that the test is good.